system-addon/content-src/components/Base/_Base.scss

```diff
@@ -1,4 +1,6 @@
 .outer-wrapper {
+  background: var(--newtab-background-color);
```

Comment:
`background` vs `background-color` does have behavior differences in that setting `background: <color>` will cause other properties of `background-*` to be overridden. We've run into bugs in the past (I don't recall exactly which) where unintended background properties were being set… but potentially there are the opposite bugs where not setting the other `background-*` properties can cause problems…